build fails for welder-web

Latest welder-web uses expat, which needs to be compiled in `release-source`. This [fails](https://fedorapeople.org/groups/cockpit/logs/release-0.1.2/log):
```
  cc '-DNODE_GYP_MODULE_NAME=expat' '-DUSING_UV_SHARED=1' '-DUSING_V8_SHARED=1' '-DV8_DEPRECATION_WARNINGS=1' '-D_LARGEFILE_SOURCE' '-D_FILE_OFFSET_BITS=64' '-DPIC' '-DHAVE_EXPAT_CONFIG_H' '-DNDEBUG' -I/tmp/home/.node-gyp/8.11.3/include/node -I/tmp/home/.node-gyp/8.11.3/src -I/tmp/home/.node-gyp/8.11.3/deps/uv/include -I/tmp/home/.node-gyp/8.11.3/deps/v8/include -I../deps/libexpat -I../deps/libexpat/lib  -fPIC -pthread -Wall -Wextra -Wno-unused-parameter -m64 -Wno-missing-field-initializers -O3 -fno-omit-frame-pointer  -MMD -MF ./Release/.deps/Release/obj.target/expat/deps/libexpat/lib/xmlrole.o.d.raw   -c -o Release/obj.target/expat/deps/libexpat/lib/xmlrole.o ../deps/libexpat/lib/xmlrole.c
  rm -f Release/obj.target/deps/libexpat/libexpat.a && ar crs Release/obj.target/deps/libexpat/libexpat.a Release/obj.target/expat/deps/libexpat/lib/xmlparse.o Release/obj.target/expat/deps/libexpat/lib/xmltok.o Release/obj.target/expat/deps/libexpat/lib/xmlrole.o
  rm -rf "Release/libexpat.a" && cp -af "Release/obj.target/deps/libexpat/libexpat.a" "Release/libexpat.a"
  g++ '-DNODE_GYP_MODULE_NAME=node_expat' '-DUSING_UV_SHARED=1' '-DUSING_V8_SHARED=1' '-DV8_DEPRECATION_WARNINGS=1' '-D_LARGEFILE_SOURCE' '-D_FILE_OFFSET_BITS=64' '-DBUILDING_NODE_EXTENSION' -I/tmp/home/.node-gyp/8.11.3/include/node -I/tmp/home/.node-gyp/8.11.3/src -I/tmp/home/.node-gyp/8.11.3/deps/uv/include -I/tmp/home/.node-gyp/8.11.3/deps/v8/include -I../../nan -I../deps/libexpat -I../deps/libexpat/lib  -fPIC -pthread -Wall -Wextra -Wno-unused-parameter -m64 -O3 -fno-omit-frame-pointer -fno-rtti -fno-exceptions -std=gnu++0x -MMD -MF ./Release/.deps/Release/obj.target/node_expat/node-expat.o.d.raw   -c -o Release/obj.target/node_expat/node-expat.o ../node-expat.cc
make[1]: g++: Command not found
make[1]: *** [node_expat.target.mk:102: Release/obj.target/node_expat/node-expat.o] Error 127
make[1]: Leaving directory '/tmp/home/build/source.jd1bi0/repo/node_modules/node-expat/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/lib/node_modules/npm/node_modules.bundled/node-gyp/lib/build.js:258:23)
gyp ERR! stack     at emitTwo (events.js:126:13)
gyp ERR! stack     at ChildProcess.emit (events.js:214:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
gyp ERR! System Linux 3.10.0-693.21.1.el7.x86_64
gyp ERR! command "/usr/bin/node.real" "/usr/lib/node_modules/npm/node_modules.bundled/node-gyp/bin/node-gyp.js" "rebuild"
```

It could be that this needs more stuff beyond installing gcc-c++, this needs to be tested.

CC @bcl

--- a/tests/Dockerfile
+++ b/tests/Dockerfile
@@ -14,11 +14,14 @@ RUN dnf -y update && \
         fedpkg \
         fontconfig \
         freetype \
+        gcc \
+        gcc-c++ \
         gettext \
         git \
         gnupg \
         grubby \
         hardlink \
+        jq \
         kernel \
         koji \
         krb5-workstation \

